products/samsungmobile.md

```diff
@@ -812,29 +815,29 @@ releases:
 -   releaseCycle: "galaxy-m21s"
     releaseLabel: "Galaxy M21s"
     releaseDate: 2020-11-06
-    eoas: false
-    eol: false
+    eoas: true
+    eol: true
     link: https://doc.samsungmobile.com/SM-F415F/INS/doc.html
 
 -   releaseCycle: "galaxy-z-fold2-5g"
     releaseLabel: "Galaxy Z Fold2 5G"
     releaseDate: 2020-11-04
     eoas: true
-    eol: false
+    eol: 2024-11-04
     link: https://doc.samsungmobile.com/SM-F916B/XEH/doc.html
 
 -   releaseCycle: "galaxy-m31-prime"
     releaseLabel: "Galaxy M31 Prime"
     releaseDate: 2020-10-17
-    eoas: false
-    eol: false
+    eoas: true
+    eol: true
     link: https://doc.samsungmobile.com/sm-m315f/ins/doc.html
 
 -   releaseCycle: "galaxy-f41"
     releaseLabel: "Galaxy F41"
     releaseDate: 2020-10-16
-    eoas: false
-    eol: false
+    eoas: true
+    eol: true
```

Comment:
```suggestion
    eoas: 2022-10-16
    eol: 2024-10-16
```

https://androidspotlight.com/software-update-policy-for-every-samsung-device/, guessed EOL